fixtures/example-app/haul.config.js

```diff
@@ -0,0 +1,7 @@
+import { createWebpackConfig } from 'haul';
+
+export default {
+  webpack: createWebpackConfig(({ platform }) => ({
```

Comment:
Unnecessary function syntax and unused argument

Reply:
It's an example

Reply:
It's a fixture, but it's still unnecessary